en/frames_plane/turbo_timber_evolution.md

```diff
@@ -0,0 +1,203 @@
+While it's possible to fly even the most simple foamboard airplane, a lot of nuance in airborne handling can be finessed with a bit more engineering effort.
+This airplane is a premium example of that, with features like frise ailerons to minimize adverse yaw.
+
+Key Build Features
+* Easy overall build with minimal airframe setup
+* Easy access to Pixhawk USB and debug connector
+* FPV with camera pan mount
+* Air data provided by wing slung pitot static pod
+* Long flight times (with Liion battery option > 24 minutes)
+
+# Build
+
+## Parts List
+
+- [Turbo Timber Evolution PNP (includes motor, servos, esc, etc, all fully installed)](https://www.horizonhobby.com/product/turbo-timber-evolution-1.5m-pnp-includes-floats/EFL105275.html#)
+- [Pixhawk 4 Mini/GPS/Power module/ set](https://shop.holybro.com/pixhawk4-mini_p1120.html)
+- SIK telemetry radio
+- MS4525DO differential pressure module and pitot tube
+- [Caddx Vista FPV air unit](https://caddxfpv.com/products/caddx-vista-kit)
+- [DJI FPV Goggles](https://www.dji.com/fpv)
+- [ExpressLRS Matek Diversity RX](http://www.mateksys.com/?portfolio=elrs-r24)
+- [Custom designed 3D printed parts](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/fw/turbo_timber_evolution/3d_printed_parts.zip)
+  - Pixhawk 4 Mini mount and top GPS mount
+  - FPV pod and camera mount
+  - Pitot static pod and wing hardpoint hanger
+- Misc hardware: M3 hardware (standoffs, washers, bolts), XT30 connector, hot glue, heatshrink, Molex Microfit connectors
+- Silicone wiring (14awg for high current, 16awg for low current, 22awg for low power and signals)
+- 3.6Ah 4S LiPo OR 4s2p 18650 LiIon
+
+## Airframe Build
+
+The TTE comes out of the box very complete as-is.
+Servos and linkages have already been installed, and the only real work remaining is finishing work, such as installing the landing gear and horizontal stabilizer.
+For this portion of the assembly, I simply followed the manual.
+
+## FPV Pod
+
+![Window and front fuselage (hatch) with FPV Pod mounted on top](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod_hatch.jpg)
+
+![Underside of hatch showing the FPV pod attachement screws and wires pulled through](../../assets/airframes/fw/turbo_timber_evolution/hatch_underside.jpg)
+
+The FPV pod was mounted on top of the battery hatch using M3 nylon hardware.
+Mounting holes for the pod were located by placing the FPV pod on top (being careful to center it with a ruler) and then punching through the fpv pod mounting holes into the foam with a screwdriver.
+A long M3 nylon screw and a washer on the underside, followed by a washer and standoff on the top of the battery hatch were then be used to mount the FPV pod.
+
+## Pitot Pod
+
+A airspeed sensor is highly recommended for use on fixed wing vehicles.
+This build uses a MS4525DO differential pressure module and pitot tube housed in a 3D printed pod that has a hardpoint hanger for connecting it to the wing.

```

Comment:
You have done something special/different here. Most people just buy and off the shelf job while you have created your housing by 3d printing - presumably to get a more professional mounting to the wing.
- What I have done is made it clear that you have done this because it isn't obvious from the parts list unless you know what a pitot is and the sensor. 
- We should perhaps say "why" - ... because this results in a better X, Y, Z.
- We should link to alternative sensor options "You can use any of the sensors here if you prefer".